contrib/opencensus-ext-azure/tests/test_transport_mixin.py

```diff
@@ -318,3 +321,18 @@ def test_transmission_400(self):
                 post.return_value = MockResponse(400, '{}')
                 mixin._transmit_from_storage()
             self.assertEqual(len(os.listdir(mixin.storage.path)), 0)
+
+    def test_transmission_307(self):
```

Comment:
Can you add a test for _MAX_CONSECUTIVE_REDIRECTS to validate circular reference is handled?

Reply:
Added.